_playwright-tests/tests/API/Repositories.spec.ts

```diff
@@ -272,4 +272,55 @@ test.describe('Repositories', () => {
       expect(resp.raw.status).toBe(204);
     });
   });
+
+  test('Verify that repository creation ignores duplicate distro versions', async ({ client }) => {
+    const repoName = randomUUID();
+    const repoUrl = 'https://content-services.github.io/fixtures/yum/comps-modules/v2/';
```

Comment:
Hi, I see you are using a URL also used in another test and then you later delete all repos with this URL. at the start and end of the test. I think this might cause problems in future if tests run in parallel. How about using a two-part name so you just delete repos that match the prefix part of the name?

Reply:
good point, at the time of writing this test the repo helpers weren't there yet 😅 
rebased and made the test use a random URL and name, so this isn't an issue anymore 👍🏼

Reply:
Thank you